plugins/mongo_db_plugin/mongo_db_plugin.cpp

```diff
@@ -408,7 +407,7 @@ void mongo_db_plugin_impl::consume_blocks() {
       _account_controls = mongo_conn[db_name][account_controls_col];
 
       while (true) {
-         boost::mutex::scoped_lock lock(mtx);
+         std::unique_lock<std::mutex> lock(mtx);
          while ( transaction_metadata_queue.empty() &&
```

Comment:
This could also be `std::lock_guard`, but again not worth changing unless you change the one above.